go/ct/st/block_context.go

```diff
@@ -23,6 +24,7 @@ import (
 type BlockContext struct {
 	BaseFee     U256       // Base fee in wei
 	BlobBaseFee U256       // Blob Base fee in wei
+	BlobHashes  []vm.Hash  // List of blob hashes
```

Comment:
I have not looked into this in detail, but as I understand the Blob system right now, blobs should be associated to the `CallContext`, not the `BlockContext`. Each transaction in a block has its own set of blobs.

Reply:
Thinking about it, it is neither part of the `BlockContext` nor of the `CallContext`. It is part of the `TransactionContext`, which is a new thing that could be added.

This would also move the generation challenge of the the constraint solver out of the `BlockContext` generator, avoiding adding additional complexity to it. Wdyt?

Reply:
well, since the the origin of `BlockContext` and `CallContext` it was mentioned both of them had info that was actually `TransactionContext`. Should we define more fields to be moved from either of those to a `TransactionContext` ? 
That should probably a separate PR. Maybe a follow-up PR and this one just adds `BlobHashes`, which are later on moved to the `TransactionContext` ?